google/cloud/spanner_v1/session.py

```diff
@@ -255,7 +266,10 @@ def delete(self):
             observability_options=observability_options,
             metadata=metadata,
         ), MetricsCapture():
-            api.delete_session(name=self.name, metadata=metadata)
+            api.delete_session(
+                name=self.name,
+                metadata=metadata,
```

Comment:
Is this missing a call to `database.metadata_with_request_id`?

Reply:
Right above on line 255 we created it, to be used inside trace_call but sure I can bring it down and inline it, since anyways trace_call's updates will be added later on independently